jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -2152,57 +2132,6 @@ YUI.add('juju-env-go', function(Y) {
     },
 
     /**
-       Change the constraints of the given service.
-
-       @method set_constraints
-       @param {String} serviceName The service name.
-       @param {Object} constraints The new service constraints.
-       @param {Function} callback A callable that must be called once the
-        operation is performed.
-       @return {undefined} Sends a message to the server only.
-    */
-    set_constraints: function(serviceName, constraints, callback) {
```

Comment:
That call wasn't used, and we'll use updateService to set constraints from now on when required.